packages/manager/src/features/CloudPulse/Alerts/AlertsResources/AlertsResources.tsx

```diff
@@ -210,7 +254,34 @@ export const AlertResources = React.memo((props: AlertResourcesProp) => {
               regionOptions={regionOptions}
             />
           </Grid>
+          {isSelectionsNeeded && (
+            <Grid item md={4} xs={12}>
+              <Checkbox
+                sx={(theme) => ({
+                  maxHeight: '34px',
+                  pt: theme.spacing(1.2),
+                  svg: {
+                    backgroundColor: theme.color.white,
+                  },
+                })}
+                data-testid="show_selected_only"
+                disabled={!(Boolean(selectedResources.length) || selectedOnly)}
```

Comment:
no need Boolean function 
```suggestion
                disabled={!selectedResources.length || selectedOnly}
```